FirebaseCombineSwift/Sources/Auth/MultiFactor+Combine.swift

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Google LLC
```

Comment:
I'd like to revisit the directory structure since I think we might prefer a product specific based directory structure like `FirebaseAuthCombine` to be consistent with the `FirebaseStorageSwift` structure already being used.

I'm fine to defer this refactor until after moving to master.

Reply:
The reason for keeping everything under one umbrella folder was to make it easier to find the code (and any related documentation) that belongs together.

I can see benefits and drawbacks for both approaches. 

On a related note, would this refactoring be part of our endeavour to reduce the number of top-level folders (to bring the README.md above the fold), as suggested by @ncooke3 ?

Reply:
Understood  thanks. Let's defer this discussion until after the merge and the README.md and top-level folder structure plan is further along that can incorporate this open question.

Reply:
I think the compelling reason to change the organization will be the installation and build time of including a dependency on all Firebase products, especially Firestore. Let's sort out after the merge.